### PR TITLE
Feature asg module

### PR DESCRIPTION
**Pull request by avmikholap**
asg module completed

### CODE REVIEW

File: terraform/environments/files_config/authn.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ - python3-pip
+ - git
+write_files:
+- content: |
+    127.0.0.1 ansible_connection=local
+  path: /tmp/inventory.ini
+  permissions: '0444'
+runcmd:
+   - pip install ansible
+   - cd /tmp/ && git clone --branch features-hw-24-26 https://github.com/CyberCuCuber/dos14-polikarpov_ruslan-gitflow.git authn
```
Comment (ggramal) [inline terraform/environments/files_config/authn.yaml]: lets make branch name settable

File: terraform/environments/files_config/authz.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - python3-pip
+  - git
+write_files:
+  - content: |
+      127.0.0.1 ansible_connection=local
+    path: /tmp/inventory.ini
+    permissions: '0444'
+runcmd:
+  - pip install ansible
+  - cd /tmp/ && git clone --branch feature-hw-26 https://github.com/avmikholap/dos14-Miholap_Aleksey_git-flow.git authz
```
Comment (ggramal) [inline terraform/environments/files_config/authz.yaml]: same as https://github.com/ggramal/dos14-omegabank-infra/pull/6/files#r1342519011

File: terraform/environments/files_config/bank.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+packages:
+ - python3-pip
+ - git
+write_files:
+- content: |
+    127.0.0.1 ansible_connection=local
+  path: /tmp/inventory.ini
+  permissions: '0444'
+- content: |
+    bank
```
Comment (ggramal) [inline terraform/environments/files_config/bank.yaml]: secret :thinking: ?

File: terraform/environments/files_config/authn.yaml (hunk 1 above)
Comment (avmikholap) [reply terraform/environments/files_config/authn.yaml]: done

File: terraform/environments/files_config/authz.yaml (hunk 2 above)
Comment (avmikholap) [reply terraform/environments/files_config/authz.yaml]: done

File: terraform/environments/files_config/bank.yaml (hunk 3 above)
Comment (avmikholap) [reply terraform/environments/files_config/bank.yaml]: done

**Review by ggramal — APPROVED** [review]
Kaif! Lets go dalshe!

**Pull request merged**